README.md

```diff
@@ -162,6 +162,7 @@ import * as t from 'io-ts'
 | refinement | ✘ | `t.refinement(A, predicate)` |
 | map | ✘ | `t.map(f, type)` |
 | prism | ✘ | `t.prism(type, getOption)` |
+| strict | ✘ | `t.strict(type)` |
```

Comment:
There is no mention that this only works on interfaces, and what it means exactly.

Reply:
I thought that the signature and the comment "specifies that only the given interface properties are allowed" would be enough, feel free to suggest any improvement though

Reply:
Right, just saw you added a JSDoc comment to the actual function. That's probably enough. You could write ``t.strict(interface)` though.